sources/packages/backend/apps/api/src/route-controllers/models/pagination.dto.ts

```diff
@@ -103,9 +103,18 @@ export class OfferingChangePaginationOptionsAPIInDTO extends PaginationOptionsAP
 }
 
 /**
- * Extended pagination option for the inprogress and completed application offering change.
+ * Extended pagination option for the completed application offering change.
  */
-export class OfferingChangePaginationRequestedOptionsAPIInDTO extends PaginationOptionsAPIInDTO {
+export class CompletedOfferingChangePaginationOptionsAPIInDTO extends PaginationOptionsAPIInDTO {
```

Comment:
Outside the PR scope but I believe that if the DTO are endpoints specifics they should be inside the DTO-specific files for those endpoints.